logchange-cli/pom.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- MY I HAVE YOUR ATTENTION PLEASE -->
+    <!-- https://repo1.maven.org/maven2/io/micronaut/micronaut-parent/3.6.1/micronaut-parent-3.6.1.pom -->
+    <!-- https://repo1.maven.org/maven2/io/micronaut/micronaut-bom/3.6.1/micronaut-bom-3.6.1.pom -->
+
+    <parent>
+        <groupId>dev.logchange</groupId>
+        <artifactId>logchange</artifactId>
+        <version>1.13.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>logchange-cli</artifactId>
+
+    <properties>
+        <packaging>jar</packaging>
+        <graal.version>21.1.0</graal.version>
+        <jdk.version>21</jdk.version>
+        <!-- If you are building with JDK 9 or higher, you can uncomment the lines below to set the release version -->
+        <release.version>21</release.version>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+
+        <maven-compiler-plugin.version>3.13.0</maven-compiler-plugin.version>
+        <native-maven-plugin.version>0.10.3</native-maven-plugin.version>
+        <maven-shade-plugin.version>3.6.0</maven-shade-plugin.version>
+
+        <micronaut.version>4.7.7</micronaut.version>
+        <micronaut.aot.version>1.1.1</micronaut.aot.version>
+        <micronaut-maven-plugin.version>3.5.4</micronaut-maven-plugin.version>
+        <picocli.version>4.6.3</picocli.version>
+        <exec.mainClass>dev.logchange.cli.LogchangeCliCommand</exec.mainClass>
+
+        <jib-maven-plugin.version>3.4.4</jib-maven-plugin.version>
+        <jib.docker.image>${project.artifactId}</jib.docker.image>
+        <jib.docker.tag>${project.version}</jib.docker.tag>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.micronaut</groupId>
+                <artifactId>micronaut-bom</artifactId>
+                <version>3.10.4</version>
```

Comment:
micronaut.version ?

Reply:
From what I can see, `micronaut-bom` and `micronaut-core` have separate versioning:  
- https://mvnrepository.com/artifact/io.micronaut/micronaut-bom  
- https://mvnrepository.com/artifact/io.micronaut/micronaut-inject  
@marwin1991